gravitee/requirements.yaml

```diff
@@ -0,0 +1,9 @@
+dependencies:
+- name: elasticsearch
+  version: ^1.22.0
+  repository: https://kubernetes-charts.storage.googleapis.com/
+  condition: es.enabled
+- name: mongodb-replicaset
+  version: ^3.9.0
+  repository: https://kubernetes-charts.storage.googleapis.com/
+  condition: mongodb.enabled
```

Comment:
@previ Please update the README to explain how to disable / enable MongoDB dependency while installing the chart (`--set es.enabled=false`) and the default value.